### ISSUE TITLE
version 3 support?

### ISSUE THREAD

**Issue by carlituxman** [ActiveCampaign/activecampaign-api-php#123]
Can't use api 3 version.

Calling to "api/3/contacts" receive this error

Class 'AC_' not found
ActiveCampaign::api
vendor/activecampaign/api-php/includes/ActiveCampaign.class.php:89

**Comment by sparkalow** [comment]
Is this project recommended?  The last commits are years old.

**Comment by pdtnelson** [comment]
Ima gonna go with "No" given that it took reading their code and coming here to open an issue for v3 support just to have it confirmed that there isn't any.

**Comment by skotfred** [comment]
Try looking at https://developers.activecampaign.com/reference/overview

**Comment by pdtnelson** [comment]
tl;dr: Don't say you maintain your SDK's when they don't work with the new major version of your API because you're wasting (developers') time and time is (their) money.

@skotfred - I think the issue at hand isn't that they don't have the new API documented well, which they do. With those docs, it's pretty easy to roll your own client and models but  for a dev who looks better to the boss when things get done quickly, it's going to be very tempting to grab that SDK and you're going to end up with a lot of lost development time if you find that out too late. 

The specific thing that has me so bothered by this situation is that https://developers.activecampaign.com/page/sdks explicitly says that this SDK is maintained by the vendor but it's out of date with a major version of their API. Ideally, the vendor would update the code but I think the very least they could do is make it more clear that this SDK will not work with the API described in their current documentation. I do appreciate you calling out that link but I just don't think it's helpful in this case.

**Comment by gavin310** [comment]
@pdtnelson Yep, just wasted a bunch of time getting the PHP SDK all set up and and reading through the examples, only to find out it doesn't even support their current API. At least edit the project's README or something. What a joke.

**Comment by gavin310** [comment]
By the way there's lots of 3rd party API v3 wrappers: https://packagist.org/?query=activecampaign

**Comment by sparkalow** [comment]
I'm going with a lack of any official response in over a year or two and a general bad taste from the community as evidence I should avoid this library and it is not supported by AC.  Don't waste your time and just unsub.
